README.md

```diff
@@ -52,7 +53,7 @@ run.config:
 ---
 
 #### postgresql_client_version
```

Comment:
"You're" was an extraneous word, but the rest of the sentence was accurate. The engine checks your Gemfile and Gemfile.lock (to catch dependencies of dependencies) to see if `pg` is listed for installation, and if so (indicating that your app will be using PostgreSQL in the first place), it installs the client, too. Otherwise, it doesn't install the client, because why bother adding packages that aren't going to be used, anyway?

Reply:
@danhunsaker 👋 okay. Should this whole thing be re-worded to something like:

  > If the 'pg' gem is detected the engine will install the postgresql client. You can specify the version to install using `postgresql_client_version` in your boxfile.yml. **Note**: the engine will try to detect the correct version first.

Reply:
I'm not certain it can autodetect the right version at all, so the Note may be incorrect... Lemme check the code.

Reply:
FWIW I was pulling that up from https://github.com/nanobox-io/nanobox-engine-ruby/pull/39/files#diff-04c6e90faac2675aa89e2176d2eec7d8R58. Figured I'd condense both sentences into one paragraph.

Reply:
Ah, right, I see it now. And have confirmed that the engine does, in fact, attempt to detect the correct version to use, so the note is accurate.

I do think having it separate helps call attention to it, so I'd leave it as two paragraphs, but otherwise, that looks like a good reword.

Reply:
👍 
a0f1c30